### ISSUE TITLE
[Accessibility]: In the Time ChartUTC DATE is automatically  converted  my local time date.

### ISSUE THREAD

**Issue by Pusendra** [carbon-design-system/carbon-charts#1131]
### Environment

Browser

### What happened? What did you expect to happen instead?

 In the Time Chart I want UTC DATE but carbon charts converts it to my local time.Can anybody suggest how not to convert local date and time while showing on graph.

### What WCAG 2.1 checkpoint does the issue violate?

   Don't know

### Version

   0.45.1

**Comment by theiliad** [comment]
Hi, you should be able to use the axis tick formatter to format the values in any way that you like https://carbon-design-system.github.io/carbon-charts/documentation/interfaces/_interfaces_axis_scales_.axisoptions.html#ticks
